lib/trino-plugin-toolkit/src/main/java/io/trino/plugin/base/ldap/JdkLdapClient.java

```diff
@@ -82,6 +85,8 @@ public JdkLdapClient(LdapClientConfig ldapConfig)
                 ldapConfig.getKeystorePassword(),
                 ldapConfig.getTrustStorePath(),
                 ldapConfig.getTruststorePassword());
+
+        this.ctxs = new HashMap<>();
```

Comment:
I don't think initialized a static variable inside constructor, since it doesn't need a complicated initial process, I think we can initialized it directly:
```
static HashMap<String, CloseableContext> ctxs = new HashMap<String, CloseableContext>();
```